tools/identity-resolution/Services/GitHubService.cs

```diff
@@ -53,21 +53,31 @@ public async Task<List<CodeownersEntry>> GetCodeownersFileEntries(Uri repoUrl)
         /// </summary>
         /// <param name="repoUrl"></param>
         /// <returns></returns>
-        private async Task<List<CodeownersEntry>> GetCodeownersFileImpl(Uri repoUrl)
+        private List<CodeownersEntry> GetCodeownersFileImpl(Uri repoUrl)
```

Comment:
I don't know this was originally coded as an async task that fetched the CODEOWNERS contents and passed those into the parser as a string. The original parser has its own file utilities to fetch the file and had an entry point that took a URL and did this. My parser needs the file as a list of string and only has the entry point that takes the absolute file path or URL. As such, I had to make this function and the calling function, GetCodeownersFileEntries, non-async.